packages/suite/src/support/messages.ts

```diff
@@ -4870,10 +4870,20 @@ export default defineMessages({
         defaultMessage: 'Application',
         description: 'Computer program.',
     },
-    TR_MAX_LABEL_LENGTH_IS: {
-        id: 'TR_MAX_LABEL_LENGTH_IS',
-        defaultMessage: 'Names can be up to {length} characters.',
-        description: 'How many characters may be in device label.',
+    TR_LABEL_REQUIREMENTS: {
+        id: 'TR_LABEL_REQUIREMENTS',
+        defaultMessage:
+            'Names can be up to {length} characters and only characters of the English alphabet are allowed.',
+        description: 'How many characters may be in device label and of what type.',
+    },
+    TR_LABEL_ERROR_LENGHT: {
+        id: 'TR_LABEL_REQUIREMENTS',
+        defaultMessage: 'Name must be {length} characters or less',
+    },
+    TR_LABEL_ERROR_CHARACTERS: {
+        id: 'TR_LABEL_REQUIREMENTS',
```

Comment:
beware of the id's, it has to be the same as the key...
```suggestion
        id: 'TR_LABEL_ERROR_LENGHT',
        defaultMessage: 'Name must be {length} characters or less',
    },
    TR_LABEL_ERROR_CHARACTERS: {
        id: 'TR_LABEL_ERROR_CHARACTERS',
```